src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/UsersController.cs

```diff
@@ -35,13 +47,28 @@ public sealed class UsersController : Controller
         };
 
         [HttpGet]
-        public ActionResult GetUsersById(Guid organisationId)
+        public async Task<ActionResult> GetUsersByOrganisationId(Guid id)
         {
-            Console.WriteLine($"Use the '{organisationId}' so the build doesn't complain.");
+            var organisation = await _organisationRepository.GetByIdAsync(id);
+            if (organisation == null)
+            {
+                return NotFound();
+            }
+
+            var organisationUsers = await _usersRepository.GetUsersByOrganisationIdAsync(id);
+            var userViewModels = organisationUsers.Select(x => new OrganisationUserViewModel
```

Comment:
No, because it's returning the result of a linq query, which will always provide a list. Granted that's knowledge that only the UsersRepository has, so from this class' perspective it could retrieve a null, but I'm relatively happy with the assumption that it won't.